tests/functional/syntax/test_for_range.py

```diff
@@ -218,9 +218,15 @@ def foo():
     for i: uint256 in range(FOO, BAR):
         pass
     """,
-        TypeMismatch,
-        "Iterator values are of different types",
-        "range(FOO, BAR)",
+        InvalidType,
+        """Expected uint256 but literal can only be cast as int128.
+  line 2:24 
```

Comment:
i don't think we need to include line numbers in error messages, we can just use the first line since the message should get regexed for

Reply:
fixed in https://github.com/vyperlang/vyper/pull/3596/commits/b4f3c39739a367e56437ac72fa3d10f9774b361e